feature/platform/tests/telemetry_inventory_test/README.md

```diff
@@ -26,23 +26,46 @@ card, power supply, disk, flash, NPU, transceiver, fabric card), validate:
 *   TODO: /components/component/software-module
 *   TODO: /components/component/software-module/state/module-type
 *   /components/component/state/description
+    *  patform_type: ["CHASSIS", "CONTROLLER_CARD", "FABRIC", "FAN", "LINECARD", "POWER_SUPPLY"]
 *   /components/component/state/firmware-version
+    *  patform_type: ["TRANSCEIVER"]
 *   /components/component/state/hardware-version
+    *  patform_type: ["CHASSIS", "CONTROLLER_CARD", "FABRIC", "LINECARD", "POWER_SUPPLY", "TRANSCEIVER"]
 *   /components/component/state/id
+    *  patform_type: ["CONTROLLER_CARD", "FABRIC", "FAN", "INTEGRATED_CIRCUIT", "LINECARD", "POWER_SUPPLY", "SENSOR"]
 *   /components/component/state/mfg-date
```

Comment:
format should look like:

```
```yaml
    /path/name/here:
    /path/name2/here:
    /path/name/platform-constrained/here:
        platform_type: ["CHASSIS"]
```

Reply:
Oh I see, thanks! I updated the format to use YAML.

A few notes:
* I removed the duplicated `/state/temperature/instant` path.
* I added a TODO for the `/components/component/storage` because this still has a "t.Skipf("Telemetry path /components/component/storage is not supported.")" in the test.